qa/integration-tests/src/test/java/io/camunda/it/auth/BasicAuthIT.java

```diff
@@ -67,10 +76,22 @@ void setUp() throws JsonProcessingException {
                 List.of(new UserEntity(1L, USERNAME, "name", "", passwordEncoder.encode(PASSWORD))),
                 null));
 
+    when(authorizationServices.search(any()))
+        .thenReturn(
+            new SearchQueryResult<>(
+                1,
+                List.of(
+                    new AuthorizationEntity(
+                        1L,
+                        AuthorizationOwnerType.USER.name(),
+                        AuthorizationResourceType.APPLICATION.name(),
+                        List.of(new Permission(PermissionType.ACCESS, Set.of("*"))))),
+                null));
+
     content =
         objectMapper.writeValueAsString(
             new UserRequest()
-                .username("demo")
+                .username("demo-".concat(UUID.randomUUID().toString()))
```

Comment:
Is this working around a test isolation issue?

Reply:
I actually think this is likely not necessary as its all mocked, initially I added this in as we were trying to create a `demo:demo` user which adds confusion with the at times default user creation. I was planning to leave it in because it doesn't hurt and also shows that this value is unique and non clashing etc